core/src/autogluon/core/utils/utils.py

```diff
@@ -272,6 +272,36 @@ def get_pred_from_proba(y_pred_proba, problem_type=BINARY):
     return y_pred
 
 
+def convert_pred_probas_to_df(pred_proba_list: list, columns: List[str], problem_type: str, index: pd.Index = None) -> DataFrame:
```

Comment:
You could use `List[numpy.typing.ArrayType]` if you want a more complete type for `pred_proba_list`

Reply:
added